pkg/pool-weighted/contracts/lbp/LBPool.sol

```diff
@@ -168,12 +181,30 @@ contract LBPool is ILBPool, WeightedPool, Ownable2Step, BaseHooks {
     }
 
     /**
-     * @notice Returns the migration router, which is used to migrate LBPs to Weighted Pools.
-     * @dev This is not required when the LBP does not use migration.
-     * @return migrationRouter Address of the migration router (i.e., one that can migrate LBPs to Weighted Pools)
+     * @notice Returns the migration parameters, which are used to migrate liquidity.
+     * @dev The migration parameters are set at deployment and cannot be changed later.
+     * @return migrationRouter The address of the migration router allowed to migrate this LBP
+     * @return lockDuration The duration for which the BPTs will be locked after migration
+     * @return bptPercentageToMigrate The percentage of the BPT to migrate from the LBP to the new weighted pool
+     * @return weightProjectToken The weight of the project token in the new weighted pool
+     * @return weightReserveToken The weight of the reserve token in the new weighted pool
      */
-    function getMigrationRouter() external view returns (address) {
-        return _migrationRouter;
+    function getMigrationParams()
```

Comment:
We might as well add these to the pool's immutable params

Reply:
Sorry, I didn't quite get this comment. These params are immutable